tt_metal/impl/dispatch/kernels/cq_prefetch.cpp

```diff
@@ -1160,7 +1160,9 @@ bool process_cmd(uint32_t& cmd_ptr,
         }
         stride = process_debug_cmd(cmd_ptr);
         break;
-
+    case CQ_PREFETCH_CMD_WAIT_FOR_EVENT:
+        stride = CQ_PREFETCH_CMD_BARE_MIN_SIZE;
+        break;
```

Comment:
pedantic: missing new line below here to match the other cases

Reply:
Updated, thanks!